Projects/NET/NET04/NET04-AWS100.md

```diff
@@ -1,28 +1,31 @@
-# NET04-AWS100 — Create three Billing Alarms
+# NET04-AWS100 — Host a simple static webpage with S3 and CloudFront
 
 ## Cloud Service Provider
 * Amazon Web Services (AWS)
 
 ## Difficulty
 Level 100 (Introductory)
 
+Estimated time: 1 hour
+
 ## Project's Author(s)
-* [Chris Nagy](https://twitter.com/chris_the_nagy)
+* [Antonio Lo Fiego](https://twitter.com/antonio_lofiego)
 
 ## Objectives
 
 ###  You need to complete the following:
 
-* Create a billing alarm for $5
-* Create a billing alarm for $25
-* Create a billing alarm for $100
+* Create an S3 bucket and upload a simple static webpage
+* Create a CloudFront distribution and use the S3 bucket as origin
+* Make sure that the bucket's content can be accessed only through the CloudFront endpoint
 
 ###  You need to answer the following: 
 
-* How many billing alarms do you get for free?
-* When a billing alarm is triggered how will you know?
-* What is the difference between a billing alarm and a cloudwatch alarm?
+* What is the benefit of using a CDN?
+* How would you explain Bucket Policies?
+* What are the ways to secure access on a CDN-fronted S3 bucket?
 
```

Comment:
"What are the ways to secure...." difficult to understand the question without prev. knowledge of OAIs in my opinion.
I think "How can you restrict public access only to the CloudFront distribution?" would be easier to understand.

Reply:
The logic was for journeyers to gauge the decision between having public access bucket served by S3 versus having an OAI, but I definitely agree that this might look more complicated.

Reply:
I see, word. In that case just adding "...secure public access..." would be actually better, you're right.